app/src/main/java/org/fossasia/openevent/app/core/organizer/detail/OrganizerDetailFragment.java

```diff
@@ -58,6 +69,41 @@ public void onStart() {
         setupRefreshListener();
         getPresenter().attach(this);
         getPresenter().start();
+
+        binding.detail.uploadProfilePic.setOnClickListener(view -> uploadImage());
+    }
+
+    public void uploadImage() {
+        Intent getIntent = new Intent(Intent.ACTION_GET_CONTENT);
+        getIntent.setType("image/*");
+
+        Intent pickIntent = new Intent(Intent.ACTION_PICK, android.provider.MediaStore.Images.Media.EXTERNAL_CONTENT_URI);
+        pickIntent.setType("image/*");
+
+        Intent chooserIntent = Intent.createChooser(getIntent, "Profile Photo");
+        chooserIntent.putExtra(Intent.EXTRA_INITIAL_INTENTS, new Intent[] {pickIntent});
+
+        startActivityForResult(chooserIntent, PICK_IMAGE);
+    }
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+
+        if (data != null && requestCode == PICK_IMAGE && resultCode == Activity.RESULT_OK) {
+            Uri targetUri = data.getData();
+            ContentResolver contentResolver = getContext().getContentResolver();
+            String type = UploadUtils.getMimeTypeFromUri(contentResolver, targetUri);
+
+            try {
+                Bitmap bitmap = BitmapFactory.decodeStream(contentResolver.openInputStream(targetUri));
+                Bitmap resizedBitmap = Bitmap.createScaledBitmap(bitmap, 500, 500, false);
+                String imageString = UploadUtils.convertBitmapToString(resizedBitmap, type);
+
+                getPresenter().uploadImage(imageString);
+            } catch (FileNotFoundException e) {
+                Timber.d(e);
```

Comment:
May be we could also show a snackbar saying that "There was a problem with the image" or equivalent ?